hugegraph-api/src/main/java/com/baidu/hugegraph/auth/WsAndHttpBasicAuthHandler.java

```diff
@@ -155,7 +151,7 @@ public void channelRead(ChannelHandlerContext ctx, Object msg) {
         private void sendError(ChannelHandlerContext ctx, Object msg) {
             // Close the connection as soon as the error message is sent.
             ctx.writeAndFlush(new DefaultFullHttpResponse(HTTP_1_1, UNAUTHORIZED))
-               .addListener(ChannelFutureListener.CLOSE);
+                .addListener(ChannelFutureListener.CLOSE);
```

Comment:
maybe we should keep the origin align?

Reply:
but the current style check specify that the code indented in multiples of 4，so we need to decide which style to use

Reply:
> but the current style check specify that the code indented in multiples of 4，so we need to decide which style to use

yep, we'll discuss this issue in the developer group, ensure the rule